posts/newmovie.pug

```diff
@@ -0,0 +1,44 @@
+extends Application
+
+block content
+	.container
+		.row
+			.col-xs-8.offset-xs-2
+				h1 new movie
+				hr
+				form(action="/newmovie" method="POST")
```

Comment:
Kevin also made a form for movie inputs, because he does the admin route, deleted this, it's not necessary